Sources/FoundationPrelude/AsyncHTTPClient.swift

```diff
@@ -0,0 +1,57 @@
+import AsyncHTTPClient
```

Comment:
Technically this dependency is pretty heavyweight, since it compiles all of NIO. Could experiment extracting it to a "live" dependency module in the future...